playwright/src/test/java/com/microsoft/playwright/TestBrowserTypeConnect.java

```diff
@@ -169,6 +169,17 @@ void shouldSendExtraHeadersWithConnectRequest() throws Exception {
     }
   }
 
+  @Test
+  void shouldSetDefaultHeadersAsUserAgentWithConnectRequest() throws Exception {
+    try (WebSocketServerImpl webSocketServer = WebSocketServerImpl.create()) {
+      try {
+        browserType.connect("ws://localhost:" + webSocketServer.getPort() + "/ws");
+      } catch (Exception e) {
+      }
+      assertTrue("Playwright".contains(webSocketServer.lastClientHandshake.getFieldValue("User-Agent")));
```

Comment:
I believe it should be the other way around: `assertTrue(fieldValue.contains("Playwright"));`

Reply:
ahh silly mistake fixed this.